proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -274,6 +274,7 @@ message RecordActivityTaskStartedRequest {
     temporal.api.workflowservice.v1.PollActivityTaskQueueRequest poll_request = 6;
     temporal.server.api.clock.v1.VectorClock clock = 7;
     temporal.server.api.taskqueue.v1.BuildIdRedirectInfo build_id_redirect_info = 8;
+    int32 stamp = 9;
```

Comment:
Please ensure all proto fields have comments on them. We've been putting more of an effort to keep the documentation standards for the codebase.

Reply:
I see you were good about this below, must have missed this one.

Reply:
done